chart/compass/charts/kyma-environment-broker/templates/tests/e2e-provisioning.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "kyma-env-broker.fullname" .}}-e2e-provisioning
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "kyma-env-broker.fullname" . }}-e2e-provisioning
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ include "kyma-env-broker.fullname" . }}-e2e-provisioning
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "kyma-env-broker.fullname" . }}-e2e-provisioning
+    namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "kyma-env-broker.fullname" . }}-e2e-provisioning
+  namespace: {{ .Release.Namespace }}
+rules:
+  - apiGroups: ["servicecatalog.k8s.io"]
+    resources: ["serviceinstances"]
+    verbs: ["get", "delete"]
+---
+apiVersion: "testing.kyma-project.io/v1alpha1"
+kind: TestDefinition
+metadata:
+  name: {{ .Chart.Name }}-e2e-provisioning
+  labels:
+    app: {{ .Chart.Name }}-e2e-provisioning-test
+    app.kubernetes.io/name: {{ .Chart.Name }}-e2e-provisioning-test
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+spec:
+  disableConcurrency: false
+  template:
+    spec:
+      serviceAccountName: {{ include "kyma-env-broker.fullname" . }}-e2e-provisioning
+      containers:
+        - name: tests
+          image: {{ .Values.global.images.containerRegistry.path }}/{{ .Values.global.images.tests.e2e_provisioning.dir }}e2e-provisioning-test:{{ .Values.global.images.tests.e2e_provisioning.version }}
+          imagePullPolicy: IfNotPresent
+          env:
+          - name: APP_BROKER_URL
+            value: 'https://{{ .Values.host }}.{{ .Values.global.ingress.domainName }}'
+          - name: APP_BROKER_PROVISION_TIMEOUT
+            value: "3h"
+          - name: APP_BROKER_PROVISION_GCP
+            value: "false"
+          - name: APP_BROKER_AUTH_USERNAME
+            value: "{{ .Values.broker.username }}"
+          - name: APP_BROKER_AUTH_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "kyma-env-broker.fullname" . }}
+                key: broker-password
+          - name: APP_RUNTIME_PROVISIONER_URL
+            value: "{{ .Values.provisioner.URL }}"
+          - name: APP_RUNTIME_UUA_INSTANCE_NAME
+            value: "uaa-issuer"
+          - name: APP_RUNTIME_UUA_INSTANCE_NAMESPACE
+            value: "kyma-system"
+          - name: APP_TENANT_ID
+            value: "3e64ebae-38b5-46a0-b1ed-9ccee153a0ae"
+          command:
+          - "bash"
+          args:
+            - "-c"
+            - |
+              MAX_RETRIES=60
+
+              # copied from registration-job
+              function wait_for_access_to_api_server() {
```

Comment:
if you have `entrypoint.sh` bash file is not easier to put all bash code in one place?